### PR TITLE
Changelog updates for 20230501

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+-------------------------
+
+- Updates to the ``dns``, ``inet``, ``it``, ``org``, ``ps``, and ``risk``
+  models.
+  (`#3082 <https://github.com/vertexproject/synapse/pull/3082>`_)
+  (`#3108 <https://github.com/vertexproject/synapse/pull/3108>`_)
+  (`#3113 <https://github.com/vertexproject/synapse/pull/3113>`_)
+
+  ``inet:dns:answer``
+    Add a ``mx:priority`` property to record the priority MX response.
```
Comment (Cisphyx) [inline CHANGELOG.rst:31]: priority 'of the' MX response

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+
+  ``inet:dns:dynreg``
+    Add a form to record the registration of a domain with a dynamic DNS
+    provider.
+
+  ``inet:proto``
+    Add a form record a network protocol name.
```
Comment (Cisphyx) [inline CHANGELOG.rst:38]: 'to' record

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+  ``inet:web:attachment``
+    Add a form to record the instance of a file being sent to a web service
+    by an account.
+
+  ``inet:web:file``
+    Deprecate the ``client``, ``client:ipv4``, and ``client:ipv6`` properties
+    in favor of using ``inet:web:attachment``.
+
+  ``inet:web:logon``
+    Remove incorrect ``readony`` markings for properties.
```
Comment (Cisphyx) [inline CHANGELOG.rst:49]: 'readony'

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+
+  ``it:app:snort:rule``
+    Add an ``id`` property to record the snort rule id.
+    Add an ``author`` property to record contact information for the rule
+    author.
+    Add ``created`` and ``updated`` properties to track when the rule was
+    created and last updated.
+    Add an ``enabled`` property to record if the rule should be used for
+    snort evaluation engines.
+    Add an ``family`` property to record the software family the rule is
```
Comment (Cisphyx) [inline CHANGELOG.rst:59]: 'a' family

File: CHANGELOG.rst [hunk 5]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+  ``ou:campaign``
+    Change the ``name`` and ``names`` secondary properties from ``str`` to
+    ``ou:campname`` types.
+
+  ``ps:contact``
+    Add a ``place:name`` to record the name of the place associated with the
+    contact.
+
+  ``risk:threat``
+    Add a ``active`` property to record the interval of time when the threat
```
Comment (Cisphyx) [inline CHANGELOG.rst:78]: 'an' active

File: CHANGELOG.rst [hunk 6]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+    cluster is assessed to have been active.
+    Add a ``reporter:published`` to record the time that a reporting
```
Comment (Cisphyx) [inline CHANGELOG.rst:80]: reporter:published 'property'

File: CHANGELOG.rst [hunk 7]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+    organization first publicly disclosed the threat cluster.
+
+  ``risk:tool:software``
+    Add a ``used`` property to record the interval when the tool is assessed
+    to have been deployed.
+    Add a ``reporter:discovered`` to record the time that a reporting
```
Comment (Cisphyx) [inline CHANGELOG.rst:86]: reporter:discovered 'property'

File: CHANGELOG.rst [hunk 8]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+    organization first discovered the tool.
+    Add a ``reporter:published`` to record the time that a reporting
```
Comment (Cisphyx) [inline CHANGELOG.rst:88]: reporter:published 'property'

File: CHANGELOG.rst [hunk 9]
```diff
@@ -7,11 +7,119 @@ Synapse Changelog
...
+  creating ``ou:campaign`` nodes.
+  (`#3082 <https://github.com/vertexproject/synapse/pull/3082>`_)
+
 Bugfixes
 --------
 - Fix an issue with the search functionality in our documentation missing
   the required jQuery library.
   (`#3111 <https://github.com/vertexproject/synapse/pull/3111>`_)
+- Unique nodes when performing multi-layer lifts on secondary properties.
+  Previously it was possible to lift the same node multiple times when
```
Comment (invisig0th) [inline CHANGELOG.rst:112]: If we're going to go into this level of detail, we should call out that it only happened when lifting without a value. The current text makes it sound like all use cases were broken. I'd also accept trimming to just `Unique nodes when performing multi-layer lifts on secondary properties without a value.` :D

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**